change/@microsoft-teams-js-ceda76cc-75d3-40de-8b4b-cb392a240254.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Decoupled prefetch of CDN resources from the initialization workflow and added timeout based fail and fallback path",
```

Comment:
>Decoupled prefetch of CDN resources from the initialization workflow and added timeout based fail and fallback path [](http://example.com/codeflow?start=14&length=115)

Consider:
"Decoupled prefetch of CDN resources from the initialization workflow and added fallback when resources could not be fetched in 1.5 seconds"

Reply:
I really like this and have made that change.